_posts/2020-07-22-cockpit-224.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cockpit 224
+author: kkoukiou
+date: 2020-07-22
+tags: cockpit linux
+slug: cockpit-224
+category: release
+summary: Cockpit Podman new features and Cockpit bug fixes
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 224.
+
+### Podman: Support checkpoint and restore operations for containers
+
+Checkpointing a container stops the container while writing the state of all processes in the container to disk.
```

Comment:
```suggestion
Checkpointing a container stops the container and writes the state of all container processes to disk.
```

Reply:
Decided to not include this commit in this release.